tests/test_terrain.py

```diff
@@ -153,6 +158,10 @@ def test_attribute_functions_against_richdem(self, attribute: str, get_test_data
         attr_richdem_rst = gu.Raster(get_test_data_path(os.path.join("richdem", f"{attribute}.tif")), load_data=True)
         attr_richdem = gu.raster.get_array_and_mask(attr_richdem_rst)[0].squeeze()
 
+        # TODO: Profile curvature is opposite sign as of RichDEM, check if this is warranted
+        if attribute == "profile_curvature":
+            attr_richdem = -attr_richdem
```

Comment:
@trchudley After homogenizing the X/Y directions and scaling factors of coefficients, this is the only difference left with RichDEM. All other outputs are consistent.
We should double-check if this is a difference due to inconsistent sign in Minar vs ZT, or an error in RichDEM.

Reply:
I just checked Minar/ZT, and both have a negative sign for profile curvature.
From the old code, it looks like I had actually removed that negative sign when implementing ZT several years ago to be consistent with RichDEM. So I think we can consider that this is a RichDEM inconsistency! :smile: